src/components/Article.js

```diff
@@ -43,22 +48,32 @@ export const Article = ({ lang, title }) => {
     window.location.hash = `/quickfacts/${lang}/${title}`
   }
 
-  if (!article) {
-    return 'Loading...'
-  }
-
   const hasImage = !!article.imageUrl
 
+  const pagerEvent = {
+    nextPage: () => {
+      const sectionLength = article.sections.length
+      const nextSection = currentSection + 1
+
+      if (nextSection < sectionLength) setCurrentSection(nextSection)
+      else setCurrentSection(0)
+    },
+    prevPage: () => {
+      const prevSection = currentSection - 1
+      if (prevSection >= 0) setCurrentSection(prevSection)
+    }
+  }
+
   return (
     <Fragment>
-      <Pager>
+      <Pager event={pagerEvent}>
         <div class='page article'>
           { hasImage && <div class='lead-image' style={{ backgroundImage: `url(${article.imageUrl})` }} /> }
           <div class={'card' + (hasImage ? ' with-image' : '')}>
             <div class='title' dangerouslySetInnerHTML={{ __html: article.title }} />
```

Comment:
The title should be the title of the article on the first page but then the title of the section.

Reply:
what should we show the description in the remaining section?

Reply:
Please show the section title as the big title and leave the description and line out of the section title pages.

Reply:
https://github.com/wikimedia/wikipedia-kaios/pull/15/commits/b6f4961ca47fc76648a4e9f449224b4552d6806c

Lead view
![Screen Shot 2019-12-03 at 1 21 57 PM](https://user-images.githubusercontent.com/2560096/70050751-fc16fd00-15cf-11ea-8cec-ac07cf7776a9.png)

Section view
![Screen Shot 2019-12-03 at 1 22 05 PM](https://user-images.githubusercontent.com/2560096/70050753-fc16fd00-15cf-11ea-9a8c-3a9650b21ace.png)